project/Scala3NightlyVersions.scala

```diff
@@ -8,7 +8,7 @@ object Scala3NightlyVersions {
    * They should come at least after the last supported scala3 version
    * otherwise there is no point to use these versions.
    */
-  def nightlyReleasesAfter(version: String): List[String] = {
+  def nightlyReleasesAfter(version: String): List[DottyVersion] = {
```

Comment:
@tgodzik I've checked how we test latest nightly and there was the same issue.
Changed the signature of this function to avoid this mess with ordering.